cmd/ocm-backplane/cloud/ssm.go

```diff
@@ -0,0 +1,150 @@
+package cloud
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/openshift/backplane-cli/pkg/cli/config"
+	bpCredentials "github.com/openshift/backplane-cli/pkg/credentials"
+	"github.com/openshift/backplane-cli/pkg/ocm"
+	logger "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var Node string
```

Comment:
If this variable is not needed outside this package, we can use a lower case `node`.  https://go.dev/ref/spec#Exported_identifiers   

Though this is the only flag we need in this package, I would suggest keeping the same convention as other packages. Example:
https://github.com/openshift/backplane-cli/blob/fd3a95700cff58ecbe94bf9e921283adc08a7b0f/cmd/ocm-backplane/cloud/console.go#L58
We can define
```
var ssmArgs struct {
	node string
}
```